src/styles/styles/_card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
```

Comment:
Same with colors. Create variables for each color, so you can change them in one place later